Comments/ClientApp/flow-typed/type-defs.js

```diff
@@ -47,6 +47,7 @@ declare type ConsultationStateType = {
 	supportsDownload: boolean,
 	reference?: string,
 	userHasSubmitted: boolean,
```

Comment:
We might want to remove userHasSubmitted as it's become deprecated

Reply:
Has been removed as per suggestion.